frame/airdrop/src/benchmarking.rs

```diff
@@ -0,0 +1,211 @@
+#![cfg(feature = "runtime-benchmarks")]
+
+use super::*;
+use crate::{
+	models::{Proof, RemoteAccount},
+	AccountIdOf, Call, Config, Pallet as Airdrop, Pallet, ProofOf, RemoteAccountOf,
+};
+use composable_support::types::{EcdsaSignature, EthereumAddress};
+use composable_traits::airdrop::AirdropManagement;
+use frame_benchmarking::{account, benchmarks, impl_benchmark_test_suite};
+use frame_support::pallet_prelude::*;
+use frame_system::{Pallet as System, RawOrigin};
+use sp_io::{crypto::Pair, ed25519, hashing::keccak_256};
+use sp_runtime::traits::One;
+use sp_std::prelude::*;
+
+pub type EthKey = libsecp256k1::SecretKey;
+pub type RelayKey = ed25519::Pair;
+
+pub type BlockNumber = u32;
+pub type Moment = u32;
+
+pub const PROOF_PREFIX: &[u8] = b"picasso-";
+pub const VESTING_STEP: Moment = 3600 * 24 * 7;
+pub const VESTING_PERIOD: BlockNumber = 48 * WEEKS;
+
+const MILLISECS_PER_BLOCK: u64 = 6000;
+const MINUTES: BlockNumber = 60_000 / (MILLISECS_PER_BLOCK as BlockNumber);
+const HOURS: BlockNumber = MINUTES * 60;
+const DAYS: BlockNumber = HOURS * 24;
+const WEEKS: BlockNumber = DAYS * 7;
+
+#[derive(Clone)]
+pub enum ClaimKey {
```

Comment:
nit: Identity is the term for this

Reply:
a81dac82ce753e0b412aa068fb496c752a54360b